BraintreeCore/src/main/java/com/braintreepayments/api/core/BraintreeClient.kt

```diff
@@ -278,6 +264,22 @@ class BraintreeClient @VisibleForTesting internal constructor(
         }
     }
 
+    /**
+     * @suppress
+     */
+    @Suppress("TooGenericExceptionCaught")
+    private fun sendGraphQLPOSTSync(
+        json: JSONObject?,
+        configuration: Configuration
+    ): Pair<String?, Exception?> = try {
```

Comment:
Same here - we might be able to use the same sealed class type here.

Reply:
Yup totally agree. This is a great callout.